README.md

```
@@ -336,6 +336,34 @@ OpenClaude supports multiple providers, but behavior is not identical across all
+    }
+  },
+  "agentRouting": {
+    "Explore": "deepseek-v4-flash",
+    "Plan": "gpt-4o",
+    "general-purpose": "gpt-4o",
+    "code-reviewer": "gpt-4o",
+    "frontend-dev": "deepseek-v4-flash",
+    "default": "gpt-4o"
+  }
+}
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)